feat(ipam): add info for reserving IPs (#3678)
scaleway/docs-content: network/ipam/index.mdx

  description: Dive into Scaleway IPAM with our quickstart guides, how-tos, tutorials and more.
---

<Alert
  sentiment="info"
  title="Reserve IPs with IPAM"
>

>   You can now reserve private IPs with IPAM, and use a reserved IP when attaching a specific resource to a Private Network. Find out more in our [dedicated documentation]((/network/ipam/how-to/reserve-ip/)).

Reviewer: ```suggestion
  You can now reserve private IPs with IPAM, and use a reserved IP when attaching a specific resource to a Private Network. [Learn more about how to reserve an IP]((/network/ipam/how-to/reserve-ip/)).
```
